packages/server/src/main/resources/application-local.properties

```diff
@@ -29,6 +29,9 @@ terarium.elasticsearch.auth-enabled=false
 ########################################################################################################################
 # Keycloak configuration
 ########################################################################################################################
+terarium.keycloak.url=https://keycloak.staging.terarium.ai
+terarium.keycloak.realm=Terarium
+terarium.keycloak.admin-realm=Terarium
```

Comment:
Double check with @dgauldie but those should not be there, you have to use the `staging` profile to do this.